Documentation/Contributing/Feature_Contribution_Process.md

```diff
@@ -58,51 +59,51 @@ If needed, the architectural document can be updated to align with any changes t
 
 Please ensure that all existing Unit Tests and any new tests are all passing prior to submission.
 
-## SDK Implementation
+### SDK Implementation
 
 Once the core interfaces and events are merged in to development, work can then be submitted for the SDK components.  Adding the concrete implementation of the feature and testing against the supported platforms and unit tests.
 
-## Example Implementation
+### Example Implementation
 
 Once the SDK components are merged, then any demo scenes or updates to the example scenes can be submitted.
 
 * Demos are for specific feature highlighting and demonstration
 * Examples are full working scene learning examples
 
-# New Feature Requirements
+## New Feature Requirements
 
 Most feature implementations can be broken down into 3 main parts:
 
 1. [The Feature Manager](#manager-implementation-requirements)
 2. [The Event Data](#event-data-implementation-requirements) (Optional)
 3. [The Feature Handler](#handler-implementation-requirements) (Optional)
 
-## Manager Implementation Requirements
+### Manager Implementation Requirements
 
 * Assembly Definitions for code outside of the `MixedRealityToolkit/_Core` folder.
-    * This ensures features are self-contained and have no dependencies to other features.
-    * This only applies to `MixedRealityToolkit` folder.
+  * This ensures features are self-contained and have no dependencies to other features.
+  * This only applies to `MixedRealityToolkit` folder.
 * Be defined using an interface found in `MixedRealityToolkit/_Core/definitions/<FeatureName>System`.
 * A feature's concrete manager implementation should inherit directly from `BaseManager` or `MixedRealityEventManager` if they will raise events.
 * A feature's concrete manager implementation should setup and verify scene is ready for that system to use in `Initialize`.
 * A feature's concrete manager should also clean up after themselves removing anything created in the scene in `Destroy`.
 * Be registered with the Mixed Reality Manager.
-    * If the feature is a core feature, this should be hard coded into the `MixedRealityManager` and added to the `MixedRealityConfigurationProfile`.
-        * This includes being able to specify a concrete implementation via dropdown using `SystemType`.
-        * Features should have a configuration profile that derives from a scriptable object.
-        * A default configuration profile located in `MixedRealityToolkit-SDK/Profiles` and be assigned in the default configuration profile for the Mixed Reality Manager
-    * If this feature is **not** a core feature, then it must be registered using the component configuration profile and implement `IMixedRealityComponent`.
+  * If the feature is a core feature, this should be hard coded into the `MixedRealityManager` and added to the `MixedRealityConfigurationProfile`.
```

Comment:
MixedRealityManager -> MixedRealityToolkit and CoreServices

Reply:
Updated